app/src/main/java/org/sopt/sample/Home/HomeFragment.kt

```diff
@@ -39,15 +40,16 @@ class HomeFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?){
         super.onViewCreated(view, savedInstanceState)
-        userService.getUser().enqueue(object : Callback<ResponseUser>{
-            override fun onResponse(call: Call<ResponseUser>, response: Response<ResponseUser>) {
+
+        musicShowService.showMusic().enqueue(object : Callback<ResponseMusicShowDTO>{
+            override fun onResponse(call: Call<ResponseMusicShowDTO>, response: Response<ResponseMusicShowDTO>) {
                 if (response.isSuccessful) {
-                    userViewModel.userList.addAll(response.body()?.data!!)
+                    musicShowViewModel.musicList.addAll(response.body()?.data!!)
 
-                    val adapter = UserAdapter(requireContext())
+                    val adapter = MusicAdapter(requireContext())
                     binding.rvHome.adapter = adapter
-                    adapter.setRepoList(userViewModel.userList)
-                    Log.e("Response", "onResponse: ${userViewModel.userList}" )
+                    adapter.setMusicList(musicShowViewModel.musicList)
+                    Log.e("Response", "onResponse: ${musicShowViewModel.musicList}" )
```

Comment:
```suggestion
                    Log.d("Response", "onResponse: ${musicShowViewModel.musicList}" )
```

대개 e는 에러가 발생했을 때 사용하는거여서 d를 사용하는게 좋아

Reply:
헉 하나 배워갑니다 !!